plugins/chat/app/services/chat/list_user_channels.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
```

Comment:
This is the new service to preload user channels, it's similar to what we had in the current_user serializer of plugin.rb